forecast_script/entrypoint-scripts/00-dependencies.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Include python3
+apt -y update; apt -y install python3 python3-pip git
+
+# Add ubuntugis
+apt -y install software-properties-common wget
+add-apt-repository ppa:ubuntugis/ppa
+apt -y update
+apt -y upgrade
+
+# Add GDAL
+apt -y install gdal-bin libgdal-dev python3-gdal
+export CPLUS_INCLUDE_PATH=/usr/include/gdal
+export C_INCLUDE_PATH=/usr/include/gdal
+pip3 install GDAL==`gdal-config --version`
+
+export PG_MAJOR_VERSION=$(pg_config --version | grep -Po '(?<=PostgreSQL )[^.]+')
+
+# Add pg_cron
+apt -y install postgresql-$PG_MAJOR_VERSION-cron
+echo "shared_preload_libraries = 'pg_cron'" >> /etc/postgresql/$PG_MAJOR_VERSION/main/postgresql.conf
```

Comment:
We can run these as an envrionmental variable `-e EXTRA_CONF="log_destination = 'stderr'\nlogging_collector = on"`

Reply:
But will it fails when postgres first started up since the extension hasn't been installed yet, so the config is not recognizable?

Reply:
The extensions are installed during the initiation of the DB. So I do not think the config will not be recognisable